internal/subshell/sscommon/env.go

```diff
@@ -0,0 +1,49 @@
+package sscommon
+
+import "github.com/ActiveState/cli/internal/constants"
+
+type EnvType int
+
+const (
+	Deploy EnvType = iota
+	Default
+)
+
+type envData struct {
+	start string
+	stop  string
+	key   string
+}
+
+var lookup = [...]envData{
+	{
+		constants.RCAppendDeployStartLine,
+		constants.RCAppendDeployStopLine,
+		"user_deploy_env",
```

Comment:
```suggestion
		"user_env",
```
Changing this would needlessly break older versions that relied on it.